kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -88,32 +79,15 @@ module.exports = BaseView.extend({
                          2) The view's model is not fetched, and the user finishes filling out the new form.
         */
         this.$el.html(this.template());
-        // Explicitly hide/display desired buttons, since we re-render the template
-        this.$(".rating-skip").hide();
-        this.$(".rating-edit").hide();
         var views_and_opts = [
-            ["star_view_quality", StarView, {el: this.$("#star-container-quality"), model: this.model, rating_attr: "quality"}],
-            ["star_view_difficulty", StarView, {el: this.$("#star-container-difficulty"), model: this.model, rating_attr: "difficulty"}],
+            ["star_view_quality", StarView, {title: "Quality", el: this.$("#star-container-quality"), model: this.model, rating_attr: "quality", label_values: this.quality_label_values}],

```

Comment:
Is "Quality" shown to the user? If so, please wrap in a `gettext` call. Same for "Difficulty".